src/objects/insights.js

```diff
@@ -123,31 +114,22 @@ export default class Insights extends AbstractCrudObject {
     })
   }
 
-  static get ActionReportTime () {
+  static get ActionReportTime() {
     return Object.freeze({
       impression: 'impression',
       conversion: 'conversion'
     })
   }
 
-  static get Breakdowns () {
+  static get Breakdowns() {
     return Object.freeze({
-      age: 'age',
-      country: 'country',
-      gender: 'gender',
-      frequency_value: 'frequency_value',
-      hourly_stats_aggregated_by_advertiser_time_zone: 'hourly_stats_aggregated_by_advertiser_time_zone',
-      hourly_stats_aggregated_by_audience_time_zone: 'hourly_stats_aggregated_by_audience_time_zone',
-      impression_device: 'impression_device',
-      place_page_id: 'place_page_id',
-      placement: 'placement',
       placement_merge_rhc: 'placement_merge_rhc',
```

Comment:
looking at the documentation: https://developers.facebook.com/docs/marketing-api/insights/breakdowns/v2.8
There are still more options for Breakdonws.

- age
- country
- gender
frequency_value
- hourly_stats_aggregated_by_advertiser_time_zone
- hourly_stats_aggregated_by_audience_time_zone
- impression_device
- place_page_id
- placement
- device_platform
- product_id
- region